types/web3/utils.d.ts

```diff
@@ -66,6 +71,7 @@ export default interface Utils {
     toDecimal(val: any): number;
     toHex(val: any): string;
     toUtf8(val: any): string;
-    toWei(val: string | number | BigNumber, unit: Unit): string | BigNumber;
+    toWei(val: string | number, unit?: Unit): string;
+    toWei(val: BigNumber, unit?: Unit): BigNumber;
```

Comment:
Reference for overloading: https://web3js.readthedocs.io/en/1.0/web3-utils.html?highlight=toWei#id77